tests/cypress/integration/actions_objects2/case_14_appearance_features.js

```diff
@@ -162,5 +167,16 @@ context('Appearance features', () => {
             cy.get('.cvat-appearance-outlinded-borders-checkbox').click();
             cy.get('#cvat_canvas_shape_3').should('have.css', 'stroke', 'rgb(224, 224, 224)'); // have CSS property stroke with the value rgb(224, 224, 224)
         });
+
+        it('"Selected opacity" slider now defines opacity level of shapes being drawn.', () => {
+            cy.interactControlButton('draw-rectangle');
```

Comment:
Could you please also check it for polygons, polyline, points, and cuboids (for polyline, points it should not affect opacity level). For cuboids, it should affect opacity level only if draw using rectangle (BTW the same for rectangles, it should affect opacity level only if draw them using standard way [not using 4 points])

Reply:
Added the appropriate steps.